standalone-node/cluster_installers/download_charts_and_images.sh

```diff
@@ -7,67 +7,32 @@ IMG_DIR=./images
 CHRT_DIR=./charts
 EXT_DIR=./extensions
 TPL_DIR=./extensions-templates
-TAR_PRX=rke2-images
+TAR_PRX=k3s-images
 TAR_SFX=linux-amd64.tar
+DOWNLOAD_ARTIFACTS=false
 
 # List of pre-downloaded docker images
 images=(
-	quay.io/jetstack/cert-manager-controller:v1.16.2
-	quay.io/jetstack/cert-manager-cainjector:v1.16.2
-	quay.io/jetstack/cert-manager-webhook:v1.16.2
-	quay.io/jetstack/cert-manager-startupapicheck:v1.16.2
-	docker.io/openpolicyagent/gatekeeper:v3.17.1
-	docker.io/openpolicyagent/gatekeeper-crds:v3.17.1
-	docker.io/curlimages/curl:8.11.0
-	docker.io/openebs/provisioner-localpv:4.2.0
-	registry.k8s.io/sig-storage/csi-resizer:v1.8.0
-	registry.k8s.io/sig-storage/csi-snapshotter:v6.2.2
-	registry.k8s.io/sig-storage/snapshot-controller:v6.2.2
-	registry.k8s.io/sig-storage/csi-provisioner:v3.5.0
-	docker.io/openebs/lvm-driver:1.6.1
-	registry.k8s.io/sig-storage/csi-node-driver-registrar:v2.8.0
-	docker.io/bitnami/kubectl:1.25.15
-	registry.k8s.io/kube-state-metrics/kube-state-metrics:v2.15.0
-	quay.io/brancz/kube-rbac-proxy:v0.19.0
-	quay.io/prometheus-operator/prometheus-operator:v0.81.0
-	quay.io/prometheus/prometheus:v3.2.1
-	docker.io/library/busybox:1.35.0
-	docker.io/library/busybox:latest
-	quay.io/prometheus-operator/prometheus-config-reloader:v0.81.0
-	quay.io/prometheus/node-exporter:v1.9.0
-	docker.io/library/telegraf:1.32-alpine
-	registry.k8s.io/nfd/node-feature-discovery:v0.17.0
-	docker.io/kubernetesui/dashboard:v2.7.0
-	docker.io/kubernetesui/metrics-scraper:v1.0.8
-	docker.io/grafana/grafana:11.6.0
-	docker.io/bats/bats:v1.4.1
+	docker.io/calico/apiserver:v3.30.0
```

Comment:
does this match with @jokuniew's PR ?

Reply:
Just to clarify that nothing here is needed in the final version